data/agents.yaml

```diff
@@ -35,6 +37,7 @@ KateLondon:
       - views.NavigationBarRegion.ConnectRegion.order: MyChannelsView,MyCampaignsView,MyHistoryView,WorkbinsView,InteractionsSearchView,ContactDirectoryView
       - views.NavigationBarRegion.ConnectRegion.activate-order: MyChannelsView,MyCampaignsView,MyHistoryView,WorkbinsView,InteractionsSearchView,ContactDirectoryView
       - interaction.web-content: GenesysWikiSection
+      - workspace.web-content: "ScapiRemoteGithub"
```

Comment:
Preferable to use by default the local sample for an offline usage.

Reply:
Ok fine, I wanted to have : 
- JohnSmith using LocalSCAPISample
- KateLondon using RemoteSCAPISample

But I'll change it to have KateLondon using the "LocalSCAPISample".

The "RemoteSCAPISample" will be provisioned in Cluster, but nobody will use it 👍